.gitignore

```diff
@@ -84,6 +84,5 @@ bower_components/
 *.log
 
 build/
-dist/
```

Comment:
@Ardeshir81 `dist/` directory will be automatically aded by `npm`. We don't store it under git.

Could you please remove `dist/` directory from commit and add it back rule for `.gitignore`?